dockstore-client/src/main/java/io/dockstore/client/cli/nested/AbstractEntryClient.java

```diff
@@ -890,7 +890,7 @@ public void checkEntryFile(String localFilePath, List<String> argsList, String d
         } else if (optContent.isPresent()) {
             DescriptorLanguage content = optContent.get();
             //no extension given
-            if (content.equals(CWL)) {
+            if (CWL.equals(content)) {
                 out("This is a CWL file.. Please put an extension to the entry file name.");
```

Comment:
It was like this, but just noticed -- neither an ellipsis nor a period. Also, "Please put an extension to the entry file name" sounds awkward.

Reply:
Hmmm, I'll grab this in the next PR